azimuth/routers/config.py

```diff
@@ -98,3 +100,11 @@ def patch_config(
     # Clear workers so that they load the correct config.
     task_manager.clear_worker_cache()
     return new_config
+
+
+def attribute_changed_in_config(
+    attribute: str, partial_config: Dict[str, Any], config: AzimuthConfig
+) -> bool:
+    return attribute in partial_config and partial_config[attribute] != config.__getattribute__(
+        attribute
+    )
```

Comment:
`__getattribute__()` is the magic method that gets called when using Python’s built-in function `getattr()`. It's very rare that we have to call the magic methods directly.
```suggestion
    return attribute in partial_config and partial_config[attribute] != getattr(config, attribute)
```